coordinator/src/main/scala/filodb.coordinator/queryplanner/MultiPartitionPlanner.scala

```diff
@@ -657,7 +670,36 @@ class MultiPartitionPlanner(partitionLocationProvider: PartitionLocationProvider
             (Some(next), ep)
         }
       }
-      execPlans
+      // Edge case with namespace across multiple partitions includes a case where the a part of time range
+      // of the original params fall in the new partition but that duration is less than the lookback. For example,
+      // Consider the timesplit happening at 1:00, the lookback of the query is 20 mins and the original query params
+      // have end date of 1:10. Then the period of 1 - 1:20 will not have any results, this is due to that fact the
+      // period 1 - 1:10 can not be served from one partition alone and needs to be computed on query service. Here
```

Comment:
> this is due to that fact the period 1 - 1:10 can not be served from one partition alone and needs to be computed on query service.

Should we callout the `getQueryTimeRanges` method? Really that just needs to be updated to drop the old "empty result at partition splits" logic.